lib/rust/ensogl/core/src/display/object/class.rs

```diff
@@ -1188,6 +1188,32 @@ impl<Host> Object<Host> for Any<Host> {
 
 
 
+// =========================
```

Comment:
This is cleaner than what it replaces, but the need for it it suggests a complex lifetime dynamic I don't fully understand.

Why is it that Sprite's display Instance needs to be unparented if the Sprite is dropped, but an Instance for any other object doesn't need this?

Reply:
This is an interesting question. You know, right now all things that you see on the screen are Sprites, so this works for everything we have. We could theoretically use this pattern in every object (like a node, that contains a lot of sprites), but I'm not yet sure what benefits it would give us. But this is an interesting potential change (definitely not part of this PR as it could be really big). CC @farmaazon, what do you think?